sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Triggers/MessageToPocoConverter.cs

```diff
@@ -61,7 +87,7 @@ private TElement DeserializeJsonObject(ServiceBusReceivedMessage message)
                     CultureInfo.InvariantCulture,
 @"Binding parameters to complex objects (such as '{0}') uses Json.NET serialization or XML object serialization. 
  1. If ContentType is 'application/json' deserialize as JSON
- 2. If ContentType is not 'application/json' attempt to deserialize using Message.GetBody, which will handle cases like XML object serialization
+ 2. If ContentType is not 'application/json' and it is a Data body message, attempt to deserialize using DataContractSerializer, which will handle cases like XML object serialization
```

Comment:
Should we consider linking out to something AMQP here?  I suspect "data body message" isn't going to mean something to a lot of people.

Reply:
I think we should add this scenario to our troubleshooting guide and link to that.

Reply:
Will need to do a separate PR once the updated TSG is merged.